config/routes.rb

```diff
@@ -32,6 +32,8 @@
     patch '/phone_setup' => 'devise/two_factor_authentication_setup#set'
 
     get '/otp/new' => 'devise/two_factor_authentication#new'
+    get '/otp/send' => 'devise/two_factor_authentication#send_code'
```

Comment:
Is this new route necessary? See my previous comment about duplicating the `otp/new` functionality.